src/checks/unordered_keys.rs

```diff
@@ -10,7 +10,7 @@ impl Default for UnorderedKeysChecker {
     fn default() -> Self {
         Self {
             keys: Vec::new(),
-            template: String::from("The {1} key should go before the {2} key"),
+            template: String::from("UnorderedKey: The {1} key should go before the {2} key"),
```

Comment:
In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the `name` field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
```rust
impl UnorderedKeysChecker {
    fn message(&self, key: &str) -> String {
      // code...
    }
}
```

Reply:
```rust
fn default() -> Self {
        Self {
            keys: Vec::new(),
            template: String::from("The {1} key should go before the {2} key"),
            name: String::from("UnorderedKey"),
         }
    }
```
Is this what you require?

Reply:
Yes